static/js/contacts.js

```diff
@@ -84,11 +84,10 @@ $(document).ready(function () {
     }
     $('.theme-btn').on('click', themePickerHandler);
     displayNavbar();
-
+    fetchVersionInfo();
     $('#new-contact-point').on('click',initializeContactForm)
```

Comment:
why do you need this call here ?

Reply:
there was an issue with the function call in the specific js files. I was testing the code with different method.
I have fixed the issue now.
Apologies for the inconvenience

Reply:
yes it is ready for review

Reply:
Can you put a screenshot of that ui tab looks like ?

Reply:
Sure. Please find the screenshot attached for your reference.
<img width="581" alt="Screenshot 2023-12-22 at 17 58 32" src="https://github.com/siglens/siglens/assets/113260364/cb0f2e2b-7730-4da9-8550-cbb076f03955">

Reply:
UI looks good to me , if backend is good (Andrew) and frontend is good (bhavana) , this can be merged